rhapsody/heatmap.py

```diff
@@ -105,6 +106,96 @@ def ranks_heatmap(ranks, microbe_metadata=None, metabolite_metadata=None,
     return hotmap
 
 
+def paired_heatmaps(ranks, microbes_table, metabolites_table, microbe_metadata,
+                    features, level, normalize='log10', color_palette='magma'):
+    '''
+    ranks: pd.DataFrame of conditional probabilities.
+        Microbes (rows) X metabolites (columns).
+    microbes_table: biom.Table
+        Microbe feature abundances per sample.
+    metabolites_table: biom.Table
+        Metabolite feature abundances per sample.
+    microbe_metadata: pd.Series
+        Microbe metadata for annotating plots
+    features: list
+        Select microbial feature IDs to display on paired heatmap.
+    level: int
+        taxonomic level for annotating clustermap.
+        Set to -1 if not parsing semicolon-delimited taxonomies.
+    normalize: str
+        Column normalization strategy to use for heatmaps. Must
+        be "log10", "z_score", or None
+    color_palette: str
+        Color palette for heatmaps.
```

Comment:
```suggestion
        Color palette for heatmaps.
    keep_only_maximal_samples : bool
        Keep only the samples where one of the selected microbes is the most abundant microbe   
```

This can help visualize the patterns, especially if there are a lot of samples